Aruba Network/aruba-os/ingest/parser.yml

```
@@ -14,11 +15,12 @@ pipeline:
       name: grok.match
       properties:
         input_field: "parse_message.message.syslog_payload"
-        pattern: "%{USER_INFO}|%{FORMAT_133121}|%{KV_PART}"
+        pattern: "%{SRC_MAC}|%{USER_INFO}|%{FORMAT_133121}|%{KV_PART}"
```

For performance optimization, I suggest putting `USER_INFO` first:
```suggestion
        pattern: "%{USER_INFO}|%{SRC_MAC}|%{FORMAT_133121}|%{KV_PART}"
```